satellite/rewards/rewards.go

```diff
@@ -69,12 +69,13 @@ const (
 type OfferStatus int
 
 const (
-	// Done is a default offer status when an offer is not being used currently
-	Done = OfferStatus(iota)
+
+	// Active is a offer status when an offer is currently being used
+	Active = OfferStatus(iota)
```

Comment:
Why are we changing this?

Reply:
@egonelbre said in our meeting that if we changed Active to the first one then I wouldn't need to set the status as active in the create offer handler because it would just default to it.

Reply:
I see. I think it's nice to have it there just to be explicit that new offers has the default behavior to be set as `Active`. It's going to be harder to see that relation later on at least for me

Reply:
I changed it back to Done being the default